jenkins_jobs/windows-hst.yml

```diff
@@ -0,0 +1,64 @@
+- job:
+    name: windows-tests
+    description: 'Main Jenkins job responsible for orchestrating tasks required to run GPII Windows tests'
+    project-type: multijob
+    node: h-0005.tor1.inclusivedesign.ca
+    properties:
+      # Required by the GitHub PR builder plugin.
+      - github:
+        url: https://github.com/GPII/gpii-app
+    triggers:
+      - gh-pr-builder
+    scm:
+      - gh-pr-scm
+    builders:
+      # Each parent multijob builder passes the Jenkins WORKSPACE environment
+      # variable to its child job as a parameter so that a common Git working
+      # directory can be used.
+      - multijob:
+          name: create-windows-vm
```

Comment:
would it make sense to prefix these job names with the parent job (e.g. windows-tests-create-vm, windows-tests-delete-vm, etc) ?

It seems the infusion-tests multijob at ci.fluidproject.org is not following that convention but all the other jobs at ci.inclusivedesign.ca are. We have not written down anything at this point so it's okay to keep as-is.